### PR TITLE
Added support for Amazon Web Pages

### PR DESCRIPTION
**Pull request by ktmurray1999**
Added support for amazon web services photos and changed the way pages are utilized.

### CODE REVIEW

**Comment by ktmurray1999** [comment]
Conflicts will be fixed soon.

**Comment by mscuthbert** [comment]
Have you tested populate_from_metadata with the new csv file you've committed?  That's my only Q.  Other than that the code looks awesome!

File: djweb/dj_comp_hist/common.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+from .models import Document, Page
+from django.db.models.signals import post_save
+
+
+DJWEB_PATH = Path(os.path.abspath(os.path.dirname(__file__)))
+DATA_BASE_PATH = Path(DJWEB_PATH.parent,"computation_hist","data","processed_pdfs")
+
+
+def get_file_path(box: int, folder: int, foldername_short:str, file_type:str,
+                  doc_id:int = None, page_id:int = None, include_base_path=True, aws_file=False):
```
Comment (srisi) [inline djweb/dj_comp_hist/common.py]: Can you replace `include_base_path=True, aws_file=False` with a param `path_type` ?
The path_type can be: 
relative (currently: include_base_path=False, aws_file=False)
absolute (currently: include_base_path=True, aws_file=False)
aws (currently: include_base_path=False, aws_file=True)

Currently, I think you'll produce errors if include_base_path=True and aws_file=True. 
Plus, I think my variable name (include_base_path) isn't clear. relative and absolute are more established names.

File: djweb/dj_comp_hist/common.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    if include_base_path:
+        path = Path(DATA_BASE_PATH, path)
+
+    if aws_file:
+        path = Path("https://s3.amazonaws.com/comp-hist/docs/", path)
+
+    return path
+
+
+def create_pages(sender, instance, **kwargs):
```
Comment (srisi) [inline djweb/dj_comp_hist/common.py]: What does create_pages do? Can you either add documentation or write a TODO for yourself? (I recall you mentioning that you weren't quite finished with one function. If this is the one, just make sure to add a note)

**Comment by srisi** [comment]
Implemented my suggested changes in the keith-path-suggestion branch. @ktmurray1999  Can you have a look and merge if you like the changes?

**Pull request merged**